tests/test_catalog.py

```diff
@@ -0,0 +1,31 @@
+import boto3
+import pytest
+from tests._utils import get_time_str_with_random_suffix
+
+import awswrangler as wr
+
+
+@pytest.fixture(scope="function")
+def random_glue_database():
+    database_name = get_time_str_with_random_suffix()
+    yield database_name
+    wr.catalog.delete_database(database_name)
+
+
+def test_create_database(random_glue_database):
```

Comment:
Nice test. I guess we don't cover the catalog_id part but we are ok

Reply:
Yeah I wasn't sure if there is any way to properly test it. Do you think it would be enough to just use the one of the current account to test the parameter itself or do you think it would only be beneficial if it was tested with cross-account access.

Reply:
Wrangler is really meant for single account use so I would say cross-account is out of scope and using the current account id is enough imo